### PR TITLE
Backend/feat:redis-cluster-pipelining

### PR DESCRIPTION
**Pull request by enigma-commits**
## Type of Change

- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement
- [ ] Refactoring
- [ ] Dependency updates

## Description
redis-cluster-pipelining

### Additional Changes

- [ ] This PR modifies the database schema (database migration added)
- [ ] This PR modifies dhall configs/environment variables

## Checklist

- [ ] I formatted the code and addressed linter errors `./dev/format-all-files.sh`
- [ ] I reviewed submitted code
- [ ] I added unit tests for my changes where possible
- [ ] I added a [CHANGELOG](/CHANGELOG.md) entry if applicable

### CODE REVIEW

**Review by AshishSaini25 — APPROVED** [review]

File: flake.nix [hunk 1]
```diff
@@ -13,7 +13,7 @@
     prometheus-haskell.inputs.haskell-flake.follows = "common/haskell-flake";
 
     euler-hs = {
-      url = "github:juspay/euler-hs/ag/open-source";
+      url = "github:enigma-commits/euler-hs/Backend/feat/redis-cluster-pipelining";
```
Comment (srid) [inline flake.nix:16]: Was this change intentional?
